lib/src/screens/send/lightning_send_confirm_page.dart

```diff
@@ -0,0 +1,325 @@
+                            .textFieldBorderTopPanelColor,
+                        suffixIcon: SizedBox(width: 36),
+                        placeholderTextStyle: TextStyle(
+                          fontSize: 16,
+                          fontWeight: FontWeight.w600,
+                          color: Theme.of(context).extension<ExchangePageTheme>()!.hintTextColor,
+                        ),
+                        textStyle: TextStyle(
+                            fontSize: 16, fontWeight: FontWeight.w600, color: Colors.white),
+                        validator: null,
+                      ),
+                      SizedBox(height: 24),
+                      if (invoice.description?.isNotEmpty ?? false) ...[
+                        BaseTextFormField(
+                          enabled: false,
+                          initialValue: "${S.of(context).description}: ${invoice.description}",
+                          textInputAction: TextInputAction.next,
+                          borderColor: Theme.of(context)
+                              .extension<ExchangePageTheme>()!
+                              .textFieldBorderTopPanelColor,
+                          suffixIcon: SizedBox(width: 36),
+                          placeholderTextStyle: TextStyle(
+                            fontSize: 16,
+                            fontWeight: FontWeight.w600,
+                            color: Theme.of(context).extension<ExchangePageTheme>()!.hintTextColor,
+                          ),
+                          textStyle: TextStyle(
+                              fontSize: 16, fontWeight: FontWeight.w600, color: Colors.white),
+                          validator: null,
+                        ),
+                        SizedBox(height: 24),
+                      ],
+                    ],
+                  ),
+                );
+              }),
+            ),
+            bottomSectionPadding: EdgeInsets.only(left: 24, right: 24, bottom: 24),
+            bottomSection: Observer(builder: (_) {
+              return Column(
+                children: <Widget>[
+                  LoadingPrimaryButton(
+                    text: S.of(context).send,
+                    onPressed: () async {
+                      try {
+                        final sdk = await BreezSDK();
+                        late BZG.SendPaymentRequest req;
+
+                        lightningViewModel.setLoading(true);
+
+                        if (invoice.amountMsat == null) {
+                          req = BZG.SendPaymentRequest(
+                            bolt11: invoice.bolt11,
+                            amountMsat: int.parse(_amountController.text) * 1000,
+                          );
+                        } else {
+                          req = BZG.SendPaymentRequest(bolt11: invoice.bolt11);
+                        }
+
+                        await sdk.sendPayment(req: req);
+
+                        lightningViewModel.setLoading(false);
```

Comment:
**Enhancement:**
we do have the `lightningViewModel` for a reason, UI Layer should be only for UI, there are some other instances of these enhancements in this PR